src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -59,6 +59,66 @@ namespace rascal {
 
   namespace internal {
 
+    /**
+     * Utility to find the periodic images of the center atoms that are within
```

Comment:
I agree with you that it could be an improvement but I think this would make the computation procedure quite redundant or fuzzy to understand. Also there are a few things that will have to be thought through very well because it might just not be worth it.